CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog
 
+* [CHANGE] hedgedhttp: bump cristalhq/hedgedhttp #418
```

Comment:
Would you mind just dropping this change? Shouldn't be necessary, as there shouldn't be any functional difference.

```suggestion
```